servers/zms/src/test/java/com/yahoo/athenz/zms/DBServiceTest.java

```diff
@@ -5170,15 +5190,6 @@ public void testGetExpiredPendingDomainRoleMembers() {
         Role role = zms.dbService.getRole(domainName, roleName, false, false, true);
         assertEquals(role.getRoleMembers().size(), 4);
 
-        // now let's expire our entries
-
```

Comment:
I assumed that this was removed because it is already covered in testProcessExpiredPendingMembers.
But in testProcessExpiredPendingMembers there's no assertions. The test just runs processExpiredPendingMembers and doesn't verify that the members were expired

Reply:
It was removed since it can't be tested with a real database. The test used the file connection where we forced the request time to be whatever we wanted (in this case we predated back 31 days). with the jdbc, the request is set when you insert the object and defaults to current time. So we can't expire anything. So while the actual lines are covered by other mock tests, the real test will not be possible with MySQL DB.